src/topology/basic.lean

```diff
@@ -426,22 +426,29 @@ end
 ### Neighborhoods
 -/
 
-/-- neighbourhood filter -/
+/-- A set is called a neighborhood of `a` if it contains an open set around `a`. The set of all
+neighborhoods of `a` forms a filter, the neighorhood filter at `a`, which is here defined as the
```

Comment:
```suggestion
neighborhoods of `a` forms a filter, the neighborhood filter at `a`, is here defined as the
```

Reply:
@robertylewis did you intend to delete the "which"?

Reply:
Oops, sorry!